lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -55,12 +55,11 @@ def annotate_plugin(span, env, status, response_headers, response_body)
 
     def trace!(span, zipkin_env, &block)
       trace_request_information(span, zipkin_env)
-      span.record(Trace::Annotation::SERVER_RECV)
+      span.kind = Trace::Span::Kind::SERVER
```

Comment:
is there a way to tell if the incoming span ID was shared (from b3 headers)? if so, add span.shared=true

Reply:
updated in https://github.com/openzipkin/zipkin-ruby/pull/141/commits/1139cc55956a570a48dca0230f25ac64ef2b7b2a